client/starwhale/core/eval/model.py

```diff
@@ -0,0 +1,476 @@
+class EvaluationJob(metaclass=ABCMeta):
+    def __init__(self, uri: URI) -> None:
+        self.uri = uri
+        self.name = uri.object.name
+        self.project_name = uri.project
+        self.sw_config = SWCliConfigMixed()
+
+    @classmethod
+    def run(
+        cls,
+        project_uri: URI,
+        model_uri: str,
+        dataset_uris: t.List[str],
+        runtime_uri: str,
+        version: str = "",
+        name: str = "",
+        desc: str = "",
+        **kw: t.Any,
+    ) -> t.Tuple[bool, str]:
+        _cls = cls._get_job_cls(project_uri)
+        return _cls.run(
+            project_uri=project_uri,
+            model_uri=model_uri,
+            dataset_uris=dataset_uris,
+            runtime_uri=runtime_uri,
+            version=version,
+            name=name,
+            desc=desc,
+            **kw,
+        )
+
+    @abstractmethod
+    def info(
+        self, page: int = DEFAULT_PAGE_IDX, size: int = DEFAULT_PAGE_SIZE
+    ) -> t.Dict[str, t.Any]:
+        raise NotImplementedError
+
+    @abstractmethod
+    def remove(self, force: bool = False) -> t.Tuple[bool, str]:
+        raise NotImplementedError
+
+    @abstractmethod
+    def recover(self, force: bool = False) -> t.Tuple[bool, str]:
+        raise NotImplementedError
+
+    @abstractmethod
+    def cancel(self, force: bool = False) -> t.Tuple[bool, str]:
+        raise NotImplementedError
+
+    @abstractmethod
+    def resume(self, force: bool = False) -> t.Tuple[bool, str]:
+        raise NotImplementedError
+
+    @abstractmethod
+    def pause(self, force: bool = False) -> t.Tuple[bool, str]:
+        raise NotImplementedError
+
+    @abstractmethod
+    def compare(self, jobs: t.List[EvaluationJob]) -> t.Dict[str, t.Any]:
+        raise NotImplementedError
+
+    @abstractmethod
+    def _get_report(self) -> t.Dict[str, t.Any]:
+        raise NotImplementedError
+
+    @classmethod
+    def _get_job_cls(
+        cls, uri: URI
+    ) -> t.Union[t.Type[StandaloneEvaluationJob], t.Type[CloudEvaluationJob]]:
+        if uri.instance_type == InstanceType.STANDALONE:
+            return StandaloneEvaluationJob
+        elif uri.instance_type == InstanceType.CLOUD:
+            return CloudEvaluationJob
+        else:
+            raise NoSupportError(f"job uri:{uri}")
+
+    @classmethod
+    def list(
+        cls,
+        project_uri: URI,
+        page: int = DEFAULT_PAGE_IDX,
+        size: int = DEFAULT_PAGE_SIZE,
+    ) -> t.Tuple[t.List[t.Dict[str, t.Any]], t.Dict[str, t.Any]]:
+        _cls = cls._get_job_cls(project_uri)
+        return _cls.list(project_uri)
+
+    @classmethod
+    def get_job(cls, job_uri: URI) -> EvaluationJob:
+        _cls = cls._get_job_cls(job_uri)
+        return _cls(job_uri)
+
+
+# TODO: Storage Class Mixed
+class StandaloneEvaluationJob(EvaluationJob):
+    def __init__(self, uri: URI) -> None:
+        super().__init__(uri)
+        self.store = EvaluationStorage(uri)
+        self._sw_config = SWCliConfigMixed()
```

Comment:
in the parent class `EvaluationJob`, it has already defined a property `self.sw_config`. Does remove `self._sw_config`?